### PR TITLE
refactor: remove redundant typing

### PR DESCRIPTION
**Pull request by pidupuis**

### CODE REVIEW

File: packages/lumx-react/src/components/progress-tracker/__snapshots__/ProgressTracker.test.tsx.snap [hunk 1]
```diff
@@ -16,18 +16,18 @@ exports[`<ProgressTracker> Snapshots and structure should render defaults 1`] =
...
   <div
     className="lumx-progress-tracker__foreground-bar"
     style={
       Object {
-        "left": "0%",
-        "right": "0%",
-        "transform": "scaleX(0)",
+        "left": "50%",
+        "right": "50%",
+        "transform": "scaleX(NaN)",
```
Comment (hchevallier) [inline packages/lumx-react/src/components/progress-tracker/__snapshots__/ProgressTracker.test.tsx.snap]: Is it normal you have a NaN here ?

File: packages/lumx-react/src/components/progress-tracker/ProgressTracker.tsx [hunk 2]
```diff
@@ -66,9 +66,11 @@ const ProgressTracker: React.FC<ProgressTrackerProps> = ({
...
     theme = DEFAULT_PROPS.theme,
     ...props
-}: ProgressTrackerProps): ReactElement => {
-    const backgroundPosition: number = children.length > 0 ? 100 / (children.length * 2) : 0;
-    const trackPosition: number = children.length > 0 ? ((100 / (children.length - 1)) * activeStep) / 100 : 0;
+}) => {
+    const childrenArray = Children.toArray(children);
+    const backgroundPosition: number = childrenArray.length > 0 ? 100 / (childrenArray.length * 2) : 0;
+    const trackPosition: number =
+        childrenArray.length > 0 ? ((100 / (childrenArray.length - 1)) * activeStep) / 100 : 0;
```
Comment (gcornut) [inline packages/lumx-react/src/components/progress-tracker/ProgressTracker.tsx]:
```suggestion
        childrenArray.length > 1 ? ((100 / (childrenArray.length - 1)) * activeStep) / 100 : 0;
```

File: packages/lumx-react/src/components/progress-tracker/__snapshots__/ProgressTracker.test.tsx.snap [hunk 3]
```diff
@@ -16,18 +16,18 @@ exports[`<ProgressTracker> Snapshots and structure should render defaults 1`] =
     className="lumx-progress-tracker__background-bar"
     style={
       Object {
-        "left": "0%",
-        "right": "0%",
+        "left": "50%",
+        "right": "50%",
```
Comment (gcornut) [inline packages/lumx-react/src/components/progress-tracker/__snapshots__/ProgressTracker.test.tsx.snap]:
```suggestion
        "left": "0%",
        "right": "0%",
```

File: packages/lumx-react/src/components/progress-tracker/__snapshots__/ProgressTracker.test.tsx.snap (hunk 1 above)
Comment (gcornut) [inline packages/lumx-react/src/components/progress-tracker/__snapshots__/ProgressTracker.test.tsx.snap]:
```suggestion
        "left": "0%",
        "right": "0%",
        "transform": "scaleX(0)",
```

**Pull request merged**